src/distribution-integration/alpm-hook.sh

```diff
@@ -2,12 +2,13 @@
 
 set -euo pipefail
 
-# enumerate files that are managed by holo-files
-holo scan --short | grep '^file:' | cut -d: -f2- | sort > /tmp/files-managed-by-holo
-
-# changed files come from stdin (but without leading slash!); find files
-# managed by Holo among these
-sort | sed 's+^+/+' | comm -12 - /tmp/files-managed-by-holo | sed 's/^/file:/' | xargs -r holo apply
-
-# cleanup
-rm /tmp/files-managed-by-holo
+# ALPM calls this script from the root directory, with respect to the
+# `--root` flag.
+export HOLO_ROOT_DIR="."
```

Comment:
I agree about `pacman --root`. You learn something new every day.

Re testing mode: It's true that `holo-files` has no testing mode *at the moment*, but it could gain one, and then things will be weird. I've been bitten by "ah, that edge case will never happen in production" a few times too often in my (as of yet short) career.